### PR TITLE
Finish create trainee

### PR DESCRIPTION
**Pull request by luongvietdung**

### CODE REVIEW

File: app/controllers/supervisor/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if @user.save
+      flash[:success] = t "application.flash.trainee_created"
+      redirect_to root_url
+    else
+      render :new
+    end
+  end
+
+  private
+
```
Comment (ThuBM) [inline app/controllers/supervisor/users_controller.rb]: remove this line

File: app/models/user.rb [hunk 2]
```diff
@@ -42,4 +45,10 @@ def authenticated? remember_token
     BCrypt::Password.new(remember_digest).is_password?(remember_token)
   end
 
+  private
+
```
Comment (ThuBM) [inline app/models/user.rb]: remove this line

**Comment by ThuBM** [comment]
OK

File: app/controllers/supervisor/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Supervisor::UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    @user.role = "trainee"
```
Comment (dieunb) [inline app/controllers/supervisor/users_controller.rb]: call_back

**Pull request merged**